socorro/unittest/external/elasticsearch/test_supersearch.py

```diff
@@ -24,11 +24,11 @@
 )
 
 # Remove debugging noise during development
```

Comment:
Well this comment lies in a module that we are aiming to slaughter in a few, so do you mind if I don't change it? :)